src/hyperion/parameters/rotation.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+import os
+
+import numpy as np
+from dodal.devices.detector import DetectorParams
+from dodal.devices.detector.det_dist_to_beam_converter import (
+    DetectorDistanceToBeamXYConverter,
+)
+from dodal.devices.zebra import (
+    RotationDirection,
+)
+from pydantic import Field
+from scanspec.core import AxesPoints
+from scanspec.core import Path as ScanPath
+from scanspec.specs import Line
+
+from hyperion.external_interaction.ispyb.ispyb_dataclass import RotationIspybParams
+from hyperion.parameters.components import (
+    DiffractionExperiment,
+    OptionalGonioAngleStarts,
+    OptionalXyzStarts,
+    RotationAxis,
+    TemporaryIspybExtras,
+    WithSample,
+    WithScan,
+)
+from hyperion.parameters.constants import CONST
+from hyperion.parameters.plan_specific.rotation_scan_internal_params import (
+    RotationHyperionParameters,
+    RotationInternalParameters,
+    RotationScanParams,
+)
+
+
+class RotationScan(
+    DiffractionExperiment,
+    WithScan,
+    OptionalGonioAngleStarts,
+    OptionalXyzStarts,
+    WithSample,
+):
+    omega_start_deg: float = Field(default=0)  # type: ignore
+    rotation_axis: RotationAxis = Field(default=RotationAxis.OMEGA)
+    shutter_opening_time_s: float = Field(default=CONST.I03.SHUTTER_TIME_S)
+    rotation_angle_deg: float
```

Comment:
Should: `rotation_angle_deg` wasn't immediately obvious to me when I looked at it. Could you add a docstring to say what it is?

Reply:
I renamed it to scan_width_deg